mesh/mesquite.cpp

```diff
@@ -1145,7 +1145,7 @@ void mfem::Mesh::MesquiteSmooth(const int mesquite_option)
 {
    mfem::MesquiteMesh msq_mesh(this);
    MsqDebug::enable(1);
-   MsqPrintError err(cerr);
+   MsqPrintError err(mfem::err.GetStream());
```

Comment:
Replace `mfem::err.GetStream() -> mfem::err`.

Reply:
Done in Veselin's update.